### PR TITLE
WEBUI-171: update the description of nuxeo-grid properties

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 1]
```diff
@@ -282,44 +282,52 @@ class Grid extends Nuxeo.Element {
...
         value: 'stretch',
       },
       /**
-       * The grid template columns. This is a string containing a space separate list of values that denote how the
-       * columns behave. Each value corresponds to a column. Valid value formats are `auto` and `*fr` (e.g. 1fr, 2fr),
-       * where fr denotes a fraction of the available width of the table. Settings this property overrides `columns`.
+       * Used for advanced control on the width of the columns. It is a string containing a space-separated list of
+       * values, where each value corresponds to a column. Valid value formats are `auto` and `*fr` (e.g. `1fr, 2fr`),
+       * where `fr` denotes a fraction of the available width of the grid. E.g. using 2fr will make a column twice as
+       * big as one using 1fr. Using auto will use the remaining space available and adapt it to the element inside the
```
Comment (semisse) [inline elements/nuxeo-grid/nuxeo-grid.js:318]: Sorry, but I just noticed this now:
```suggestion
       * big as one using 1fr. Using `auto` will use the remaining space available and adapt it to the element inside the
```

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 2]
```diff
@@ -282,44 +282,52 @@ class Grid extends Nuxeo.Element {
...
         reflectToAttribute: true,
       },
       /**
-       * The grid template rows. This is a string containing a space separate list of values that denote how the
-       * rows behave. Each value corresponds to a row. Valid value formats are `auto` and `*fr` (e.g. 1fr, 2fr),
-       * where fr denotes a fraction of the available height of the table. Settings this property overrides `rows`.
+       * Used for advanced control on the height of the rows. It is a string containing a space-separated list of
+       * values, where each value corresponds to a row. Valid value formats are `auto` and `*fr` (e.g. `1fr, 2fr`),
+       * where `fr` denotes a fraction of the available height of the grid. E.g. using 2fr will make a row twice as
+       * big as one using 1fr. Using auto will use the remaining space available and adapt it to the element inside the
```
Comment (semisse) [inline elements/nuxeo-grid/nuxeo-grid.js:329]: same here:
```suggestion
       * big as one using 1fr. Using `auto` will use the remaining space available and adapt it to the element inside the
```

**Review by semisse — APPROVED** [review]

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 3]
```diff
@@ -262,14 +262,14 @@ class Grid extends Nuxeo.Element {
   static get properties() {
     return {
       /**
-       * Number of columns. Will be ignored if templateColumns is defined.
+       * Number of columns. Will be ignored if `templateColumns is defined.
```
Comment (andreacornaglia) [inline elements/nuxeo-grid/nuxeo-grid.js:265]: i remember working with jsdocs but i am not sure what does the ` and `` mean, would you mind adding a link here to the doc / brief explanation? Thanks!

Comment (semisse) [reply elements/nuxeo-grid/nuxeo-grid.js:265]: For sure we are missing the end of the backtick here

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 4]
```diff
@@ -262,14 +262,14 @@ class Grid extends Nuxeo.Element {
...
        */
       columns: {
         type: Number,
         reflectToAttribute: true,
       },
       /**
-       * Number of rows. Will be ignored if templateRows is defined.
+       * Number of rows. Will be ignored if `templateRows is defined.
```
Comment (semisse) [inline elements/nuxeo-grid/nuxeo-grid.js:272]: And also here

Comment (richardsd) [reply elements/nuxeo-grid/nuxeo-grid.js:272]: Yes, it seems that IDEA is removing my backticks 🤦

File: elements/nuxeo-grid/nuxeo-grid.js (hunk 3 above)
Comment (richardsd) [reply elements/nuxeo-grid/nuxeo-grid.js:265]: The goal is to have Studio Designer loading this info and displaying it in a markdown element. That is the reason to be using the backtick. It is not to have string literals here. Am I right Sam?

Comment (semisse) [reply elements/nuxeo-grid/nuxeo-grid.js:265]: Yes, we are leveraging JSDocs here in Studio Designer so we can display this info in our contextual help tooltips

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 5]
```diff
@@ -282,44 +282,52 @@ class Grid extends Nuxeo.Element {
...
         value: 'stretch',
       },
       /**
-       * The grid template columns. This is a string containing a space separate list of values that denote how the
-       * columns behave. Each value corresponds to a column. Valid value formats are `auto` and `*fr` (e.g. 1fr, 2fr),
-       * where fr denotes a fraction of the available width of the table. Settings this property overrides `columns`.
+       * Used for advanced control on the width of the columns. It is a string containing a space-separated list of
+       * values, where each value corresponds to a column. Valid value formats are `auto` and `*fr` (e.g. `1fr, 2fr`),
+       * where `fr` denotes a fraction of the available width of the grid. E.g. using 2fr will make a column twice as
+       * big as one using 1fr. Using ``auto will use the remaining space available and adapt it to the element inside
```
Comment (mnixo) [inline elements/nuxeo-grid/nuxeo-grid.js:318]:
```suggestion
       * big as one using 1fr. Using `auto` will use the remaining space available and adapt it to the element inside
```

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 6]
```diff
@@ -282,44 +282,52 @@ class Grid extends Nuxeo.Element {
...
         reflectToAttribute: true,
       },
       /**
-       * The grid template rows. This is a string containing a space separate list of values that denote how the
-       * rows behave. Each value corresponds to a row. Valid value formats are `auto` and `*fr` (e.g. 1fr, 2fr),
-       * where fr denotes a fraction of the available height of the table. Settings this property overrides `rows`.
+       * Used for advanced control on the height of the rows. It is a string containing a space-separated list of
+       * values, where each value corresponds to a row. Valid value formats are `auto` and `*fr` (e.g. `1fr, 2fr`),
+       * where `fr` denotes a fraction of the available height of the grid. E.g. using 2fr will make a row twice as
+       * big as one using 1fr. Using `auto will use the remaining space available and adapt it to the element inside the
```
Comment (mnixo) [inline elements/nuxeo-grid/nuxeo-grid.js:329]:
```suggestion
       * big as one using 1fr. Using `auto` will use the remaining space available and adapt it to the element inside the
```

File: elements/nuxeo-grid/nuxeo-grid.js (hunk 5 above)
Comment (richardsd) [reply elements/nuxeo-grid/nuxeo-grid.js:318]: 🤦 as I said, IDEA is f*** this everytime I use `
Thanks

**Review by mnixo — APPROVED** [review]

**Review by andreacornaglia — APPROVED** [review]

**Pull request merged**